gradoop-flink/src/main/java/org/gradoop/flink/model/api/tpgm/TemporalGraphOperators.java

```diff
@@ -41,6 +42,26 @@
   // Unary Operators
   //----------------------------------------------------------------------------
 
+  /**
+   * Compares two snapshots of this graph. Given two temporal predicates, this operation
+   * will check if a graph element (a vertex or an edge) was added, removed or kept in the snapshot
+   * based on the second predicate.
+   *
+   * A property with key {@code _diff} will be set on each graph element, its value will be set to
```

Comment:
Use the constant instead of {@code _diff} so the user knows that there is a constant for the key. Same below.